src/Simplify.elm

```diff
@@ -5127,6 +5137,204 @@ subAndCmdBatchChecks moduleName { lookupTable, parentRange, fnRange, firstArg }
 
 
 
+-- HTML.ATTRIBUTES
+
+
+htmlAttributesClassListFalseElementError : { message : String, details : List String }
+htmlAttributesClassListFalseElementError =
+    { message = "In a Html.Attributes.classList, a tuple paired with False can be removed"
+    , details = [ "You can remove the tuple list element where the second part is False." ]
+    }
+
+
+htmlAttributesClassListChecks : CheckInfo -> List (Error {})
+htmlAttributesClassListChecks checkInfo =
+    let
+        listArg : Node Expression
+        listArg =
+            checkInfo.firstArg
+
+        getTupleWithSecond : Bool -> Node Expression -> Maybe { first : Node Expression, range : Range }
+        getTupleWithSecond bool expression =
+            case Node.value expression of
+                Expression.TupledExpression (first :: second :: []) ->
+                    if getBool bool checkInfo.lookupTable second then
+                        Just { range = Node.range expression, first = first }
+
+                    else
+                        Nothing
+
+                _ ->
+                    Nothing
+
+        singleElementListChecks : { a | element : Node Expression } -> List (Error {})
+        singleElementListChecks single =
+            case getTupleWithSecond True single.element of
+                Just tuple ->
+                    singleTrueChecks tuple
+
+                Nothing ->
+                    case getTupleWithSecond False single.element of
```

Comment:
If `getTupleWithSecond`/`getBool` returned a `Maybe Bool` with the boolean value, we could remove the number of similar computations and simplify the branching.

```elm
case getBoolInSecond single.element of
  Just True -> singleTrueChecks tuple
  Just False -> [ Rule.errorWithFix ... ]
  Nothing -> []
```